tony-core/src/main/java/com/linkedin/tony/TonyJobMetadata.java

```diff
@@ -5,31 +5,28 @@
 package com.linkedin.tony;
 
 import com.linkedin.tony.util.Utils;
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
 import org.apache.hadoop.conf.Configuration;
 
 
 public class TonyJobMetadata {
-  private static final Log LOG = LogFactory.getLog(TonyJobMetadata.class);
   private String id;
   private String url;
-  private long started;
-  private long completed;
+  private Long started;
+  private Long completed;
```

Comment:
Why did you change these from `long` to `Long`? It's preferable to keep everything as a primitive `long` to avoid the implicit boxing/unboxing.

You probably changed them to object `Long` to support `null`, but another way to handle this is to use `-1L` to indicate a job does not have a start or end time.

Reply:
Yeah we can do that too.